packages/apps/src/SideBar/index.tsx

```diff
@@ -19,27 +18,15 @@ import Item from './Item';
 import NodeInfo from './NodeInfo';
 import getLogo from './logos';
 
-type Props = I18nProps;
-
-type State = {
-  isCollapsed: boolean
+type Props = I18nProps & {
+  isCollapsed: boolean,
+  collapse: () => void
 };
 
-class SideBar extends React.PureComponent<Props, State> {
-  state: State;
-
-  constructor (props: Props) {
-    super(props);
-
-    const state = store.get('sidebar') || {};
-    this.state = {
-      isCollapsed: false,
-      ...state
-    };
-  }
+class SideBar extends React.PureComponent<Props> {
 
   render () {
-    const { isCollapsed } = this.state;
+    const { isCollapsed } = this.props;
 
     return (
       <div className={`apps--SideBar ${isCollapsed ? 'collapsed' : 'expanded'}`}>
```

Comment:
Related to the comment on Apps.tsx. It should be enough to add the `'collapsed' : 'expanded'` classes once on the outermost Application wrapper.